Documentation/build-parameters.rst

```diff
@@ -0,0 +1,98 @@
+Build Parameter Reference
+=========================
+
+**For Developers:** if you add a CMake parameter, make sure to document it here.
+
+For a good overview and easy, we recommend the use of ``ccmake``. To do so, simply replace the use of ``cmake`` by ``ccmake``.
+(i.e. add a "c" in front of the command)
+Alternatively, you may also modify the ``CMakeCache.txt`` in your build directory directly (for it to exist, ``cmake`` needs to be run first).
+
+Equation-specific parameters
+----------------------------
+
+The following parameters will alter the name of the SeisSol executable.
+You may explicitly compile and install multiple of these configurations at the same time—however, you will have to re-do the CMake and build process for each configuration.
+
+- ``CMAKE_BUILD_TYPE``: Either
```

Comment:
Remove Either? As it is not in line with the description given for the other parameters..